packages/wallet/api/src/lib-wallet.js

```diff
@@ -1731,6 +1731,9 @@ export function makeWallet({
 
   const firstPathToLookup = createRootLookups();
 
+  /** @type {ReturnType<typeof makeMarshal>} */
+  const marshaller = harden({ unserialize: hydrate, serialize: dehydrate });
```

Comment:
nifty

Reply:
Yeah; we already had code to do wallet-specific marshaling.

The next step is to refine it to use board ids for things like brands and issuers so that we can tell when they're the same as brands that we get from `agoricNames` and such.